compose/material3/material3/src/commonMain/kotlin/androidx/compose/material3/TimePicker.kt

```diff
@@ -1586,10 +1586,16 @@ private enum class LayoutId {
     Selector, InnerCircle,
 }
 
+// TODO(https://github.com/JetBrains/compose-multiplatform/issues/3373) revert to expect get()
 @OptIn(ExperimentalMaterial3Api::class)
-internal expect val defaultTimePickerLayoutType: TimePickerLayoutType
+internal val defaultTimePickerLayoutType: TimePickerLayoutType
     @Composable
-    @ReadOnlyComposable get
+    @ReadOnlyComposable get() = defaultTimePickerLayoutType()
+
+@Composable
+@ReadOnlyComposable
+@OptIn(ExperimentalMaterial3Api::class)
+internal expect fun defaultTimePickerLayoutType() : TimePickerLayoutType
```

Comment:
Could you please update android file as well? I know it isn't compiling in our fork, but it will greatly simplify future sync with upstream.

Reply:
Issue with expect composable getter should be fixed i guess and this should be rolled back to it insted of upstreaming. Like in [this](https://github.com/JetBrains/compose-multiplatform-core/blob/jb-main/compose/material3/material3/src/commonMain/kotlin/androidx/compose/material3/TimeFormat.kt) file. Is is also [not updated](https://github.com/JetBrains/compose-multiplatform-core/blob/jb-main/compose/material3/material3/src/androidMain/kotlin/androidx/compose/material3/TimePicker.android.kt) in android source set.

But if you sure android source set should be modifierd i can made it

Reply:
@igordmn (the mentioned change was yours) Do we update android files in such situations? Was it just missed change or conscious choice?

Reply:
> Was it just missed change or conscious choice?

It was a conscious choice to not think about Android back then, but after that merge, we decided to move towards fixing the Android target or/and make changes easier to upstream. 

So, now it is better to write Android actual's as well, at least adding `TODO()` in them.

> to it instead of upstreaming

This kind of fixes can be upstreamed also, if the Composable getter is hard/too long to fix

Reply:
Added Android actual